app/views/layouts/_header.html.erb

```diff
@@ -12,8 +12,10 @@
           </button>
           <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
             <%= link_to "Edit Profile", edit_users_path, class: "dropdown-item" %>
-            <% if current_user.is_a?(CasaAdmin) %>
+            <% if current_user.is_a?(CasaAdmin) || current_user.is_a?(Supervisor) %>
```

Comment:
I would like permissions logic to live in a Policyfile instead of an erb, could you move it? :)